datahub-web-react/src/app/entity/shared/tabs/Dataset/Validations/acrylTypes.tsx

```diff
@@ -0,0 +1,21 @@
+import { Assertion, AssertionType } from '../../../../../../types.generated';
+
+export type AssertionStatusSummary = {
+    passing: number;
+    failing: number;
+    erroring: number;
+    total: number; // Total assertions with at least 1 run.
+    totalAssertions: number;
+};
```

Comment:
I think we can move this to a `types.tsx` file in this same folder, and skip copying over `AssertionGroup` type below. This way we can also avoid having this `acrylTypes.tsx` file on OSS

Also should open a PR on `datahub-fork` to duplicate this change over so that OSS mirroring our SaaS version.